packages/frontend/Dockerfile

```
@@ -1,7 +1,12 @@
 FROM node:21-alpine3.18 as build
 WORKDIR /app
 COPY ./ /app/
-
 RUN npm ci
 RUN npm run frontend:build
-CMD ["npm", "run", "frontend:start"]
+
+FROM node:21-alpine3.18 as run
+WORKDIR /app
+
+COPY --from=build /app/packages/frontend/build/standalone ./
+
+CMD ["npm", "run", "--prefix", "./packages/frontend", "startstandalone"]
```

```suggestion
ENV PORT 80

CMD node server.js
```

This way we can get rid of the `startstandalone` command in the package.json

---

Changed the Command entirely. ENV is not required.